db-init.js

```diff
@@ -0,0 +1,35 @@
+import { DynamoDB } from "@aws-sdk/client-dynamodb"
+
+const { NODE_ENV, SETUP_DB=false } = process.env.NODE_ENV;
+
+if (!SETUP_DB)
+    throw new Error("The database should be set to true!")
```

Comment:
Maybe we should be a bit more descriptive in our error message, something like "To init the database, you must set the 'SETUP_DB' env variable to 'true' "